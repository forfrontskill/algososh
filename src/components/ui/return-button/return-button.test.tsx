import renderer from 'react-test-renderer';
import { ReturnButton } from './return-button';

describe('Test ReturnButton component', () => {

    it('ReturnButton rendering', () => {
        const tree = renderer
            .create(<ReturnButton  label='test'/>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});