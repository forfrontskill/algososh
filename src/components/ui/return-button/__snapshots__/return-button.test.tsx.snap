// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ReturnButton component ReturnButton rendering 1`] = `
<button
  className="button "
  label="test"
  type="button"
>
  <svg
    fill="none"
    height="24"
    viewBox="0 0 24 24"
    width="24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clipRule="evenodd"
      d="M12.7073 4.29289C13.0978 4.68342 13.0978 5.31658 12.7073 5.70711L7.41436 11H19.0002C19.5524 11 20.0002 11.4477 20.0002 12C20.0002 12.5523 19.5524 13 19.0002 13H7.41436L12.7073 18.2929C13.0978 18.6834 13.0978 19.3166 12.7073 19.7071C12.3167 20.0976 11.6836 20.0976 11.293 19.7071L3.58594 12L11.293 4.29289C11.6836 3.90237 12.3167 3.90237 12.7073 4.29289Z"
      fill="#0032FF"
      fillRule="evenodd"
    />
  </svg>
  <p
    className="text text_type_button text_color_link ml-4"
  >
    К оглавлению
  </p>
</button>
`;
